### DISCUSSION TITLE
Some fledgling bindings for remark in ReScript

### DISCUSSION THREAD

**Discussion by ghost** [remarkjs/remark#792] [Show and tell]
FYI, in case anybody finds this useful: https://github.com/ocaml/v3.ocaml.org/blob/54a47ab0de6800d517131da793d82f1c852acf2e/src/Unified.res

**Comment by ChristianMurphy** [comment]
Thanks for sharing @kanishka-work!
It's also worth noting that the ReScript community is working on interop for generating bindings based off TypeScript definitions
https://forum.rescript-lang.org/t/possible-to-import-typescript-packages-into-reason/989 and https://github.com/jsiebern/re-typescript

Much of remark/unified already has, or soon will have, TypeScript definitions which could be used to assist with generating bindings.
